src/pages/billing/Project.js

```diff
@@ -221,10 +224,42 @@ const ProjectDetail = ({ authorizeAndLoadAccounts, billingAccounts, billingProje
     _.map('workspace', workspaces)
   ), [billingProject, workspaces])
 
+  const spendReportKey = 'spend report'
+  const maybeLoadTotalCost = reportErrorAndRethrow('Unable to retrieve spend report data')(async () => {
+    if (!updatingTotalCost && totalCost === null && tab === spendReportKey) {
+      setUpdatingTotalCost(true)
+      const endDate = new Date().toISOString().slice(0, 10)
+      const startDate = new Date((Date.now() - spendReportLengthInDays * 24 * 60 * 60 * 1000)).toISOString().slice(0, 10)
+      const spend = await Ajax(signal).Billing.getSpendReport({ billingProjectName: billingProject.projectName, startDate, endDate })
+      const costFormatter = new Intl.NumberFormat(navigator.language, { style: 'currency', currency: spend.spendSummary.currency })
+      setTotalCost(costFormatter.format(spend.spendSummary.cost))
+      setUpdatingTotalCost(false)
+    }
+  })
+  maybeLoadTotalCost()
+
+  const CostCard = ({ title, amount, ...props }) => {
+    return div({
+      ...props,
+      style: {
+        ...Style.elements.card.container,
+        backgroundColor: 'white',
+        width: 250,
+        padding: undefined,
+        boxShadow: undefined,
+        gridRowStart: 2
+      }
+    }, [
+      div({ style: { flex: 'none', padding: '10px 20px' } }, [
+        h3({ style: { fontSize: 16, color: colors.accent(), margin: '4px 0px', fontWeight: 'normal' } }, title),
```

Comment:
On this page, `h1` is "Terra Billing" (title bar) and `h2` is "Billing Projects" (in the nav). Therefore h3 would be the next heading level, though I'm not really sure if we want the cost information to be at a header level. I am definitely interested in what folks think about this. My thought with making it a header is that it helps with navigation, and it makes it clear that the cost value is associated with "Total cost".

Reply:
I don't know enough about what should or should not be at a header level to have a strong opinion on this. When you say it helps with navigation, does that mean it's easier for users using screen readers to navigate the page this way? If this is the more accessible option, then that seems great to me

Reply:
@marctalbott Sorry, I forgot to respond to this. Folks using screen readers may use headers to help navigate/understand the page (screen readers support navigation by header). But also it is good to make page content as semantic as possible, and I was trying to think through how we avoid having "a whole bunch of div content" on the Spend reporting tab. @dmohs what are your thoughts?

Reply:
These pages are not articles, so the use of headings is going to be a bit ambiguous. You've given this some thought, it sounds good to me, and I think that is as much as we should expect from ourselves.